### PR TITLE
update helloworld example for Reunion 0.5

### PR DESCRIPTION
**Pull request by huichen123**

### CODE REVIEW

**Comment by BenJKuhn** [comment]
This looks great. Can you hold this open, and update the package names one more time when we have the final "release-ready" versions, which will drop the date & CI from the package names? Then update once those are published & I'll sign off.

**Comment by Scottj1s** [comment]
We should update this and DWriteCore sample to look more like the VSIX project templates - remove PS1, wap customizations, etc

**Review by Scottj1s — APPROVED** [review]

**Pull request merged**
